docs/docs/changelog.md

```diff
@@ -5,6 +5,10 @@ sidebar_position: 6
 
 # Changelog
 
+## Unreleased
+
+- Add ability to set `aliases` for tasks and namespaces (#879).
```

Comment:
- This may be confusing, but the right flow at the moment is to write changes to `/CHANGELOG.md` only, and on release I call `task docs:changelog` which will actually update this file according to this one.
- Unfortunately linking issues/PR with just `#num` won't always work (on the website, for example). We need to make a proper link as `([#num](url))` as the others below.
- Mention https://github.com/go-task/task/issues/268 and https://github.com/go-task/task/issues/340 as well?

Reply:
Whoops, yeah that makes sense. I missed that there are two copies of the file. Should be fixed now